bin/asl-setup-dkms-mok

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
```

Comment:
This is a general comment.  I don't believe that "mok" is a term that is familiar to the masses.  Might we consider changing the name of this command to something others might more readily latch on to?  Maybe something like asl-setup-secure-boot?

Also, is there a reason not to make this part of asl3-asterisk so that we always check for UEFI and Secure Boot.  If both are present, just "do it" ?

Reply:
This script is explicitly and only for the Machine Owner Key (MOK). This is not about setting up SecureBoot. Using SecureBoot with UEFI is really an OS installation issue. Honestly this whole topic is "you'll know what this means when you have this problem" kind issue.